packages/angular_devkit/schematics/testing/schematic-test-runner.ts

```diff
@@ -85,6 +85,11 @@ export class SchematicTestRunner {
       .pipe(map(tree => new UnitTestTree(tree)));
   }
 
+  /**
+   * @deprecated Use `runSchematicAsync`.  All schematics can potentially be async. This
+   * synchronous variant will fail if the schematic, any of its rules, or any schematics
+   * it calls are async.
+   */
```

Comment:
Maybe we should point out in which version these will be removed? 

It will also be useful for us when cutting a major release to know if we should remove it or not since these I guess will be removed in version 10 correct?

Reply:
Added a "deprecated since" version.  This will land in 8.0.0 so they can probably go 9.0.